### ISSUE TITLE
生成语音时会多一句「无关」内容

### ISSUE THREAD

**Issue by codingDuan** [MoonshotAI/Kimi-Audio#87]
`sampling_params = {
    "audio_temperature": 0.0,
    "audio_top_k": 10,
    "text_temperature": 0.0,
    "text_top_k": 5,
    "audio_repetition_penalty": 0.0, # 改为了0
    "audio_repetition_window_size": 0, # 改为了0
    "text_repetition_penalty": 0.0, # 改为了0
    "text_repetition_window_size": 0, # 改为了0
}

messages = [
    #{"role": "user", "message_type": "text", "content": "请将文字内容转换为音频。"},
    {"role": "user", "message_type": "audio", "content": "/aistudio/workspace/input.wav"}, #input.wav的内容为：请将文字内容转换为音频。
    {"role": "user", "message_type": "text", "content": "今天是2025.05.12,天气晴朗,阳关明媚"}
]

wav, text = model.generate(messages, **sampling_params, output_type="both")
output_audio_path = "output_audio_new.wav"
sf.write(output_audio_path, wav.detach().cpu().view(-1).numpy(), 24000) # Assuming 24kHz output
print(f">>> Conversational Output Audio saved to: {output_audio_path}")
print(">>> Conversational Output Text: ", text)
compute_end_time = time.time()
compute_elapsed_time = compute_end_time - end_time`

### 输出的output_audio_new.wav 语音会多一句「天气晴朗，阳光明媚」，然后才是「今天是xxxx」
